string sender
uint32 commandid
float64 payloadfloat
int64 payloadint
---
